frontend/pages/index.page.tsx

```diff
@@ -59,111 +89,66 @@ export default function Home() {
 
   return (
     <Layout>
-      <Grid container spacing={2} sx={{ p: 4 }}>
-        <Grid item xs={12}>
-          <Container>
-            <Box
-              sx={{
-                flex: "1 1",
-                display: "flex",
-                alignItems: "center",
-                justifyContent: "center",
-                margin: "1.5rem 1rem 1.5rem 1rem",
-              }}
-            >
-              <Card elevation={5}>
-                <CardContent>
-                  <Container>
-                    <p>Search modules by topic, entity, or keywords.</p>
-                    {searchForm}
-                  </Container>
-                </CardContent>
-              </Card>
-            </Box>
-          </Container>
-        </Grid>
-      </Grid>
-      <Grid
-        container
-        spacing={{ xs: 2, md: 3 }}
-        columns={{ xs: 12 }}
-        alignItems={"center"}
-        justifyContent={"center"}
-      >
-        <Grid item xs={12} sm={6} md={6} alignItems="center" justifyContent="center">
-          <Container>
+      <Grid container justifyContent="center" spacing={{ xs: 2, md: 3, lg: 4 }} sx={{ p: 4 }}>
+        <GridItem>
+          <StyledCard elevation={5} sx={{ maxWidth: { lg: "50rem" } }}>
+            <StyledCardHeader title="About ModularHistory" />
             <AboutModularHistory />
-          </Container>
-        </Grid>
-        <Grid item xs={12} sm={6} md={6}>
-          <Container>
-            <FeaturedContent />
-          </Container>
-        </Grid>
-      </Grid>
-      <Grid container spacing={2} sx={{ p: 4 }}>
-        <Grid item xs={12}>
-          <Card elevation={5}>
+          </StyledCard>
+        </GridItem>
+        <GridItem>
+          <StyledCard elevation={5}>
+            <StyledCardHeader title="Module Search" />
             <CardContent>
-              <TodayInHistory />
+              <p style={{ textAlign: "center" }}>Search modules by topic, entity, or keywords.</p>
+              {searchForm}
             </CardContent>
-          </Card>
-        </Grid>
-        <Grid item xs={12}>
-          <SubscriptionBox />
-        </Grid>
+          </StyledCard>
+        </GridItem>
+        <GridItem>
+          <StyledCard elevation={5}>
+            <StyledCardHeader title="Featured Content" />
+            <FeaturedContent />
+          </StyledCard>
+        </GridItem>
+        <GridItem>
+          <StyledCard elevation={5} sx={{ minWidth: "20rem" }}>
+            <StyledCardHeader title="Today in History" />
+            <TodayInHistory />
+          </StyledCard>
+        </GridItem>
+        <GridItem>
+          <StyledCard elevation={5} hidden>
+            <SubscriptionBox />
+          </StyledCard>
```

Comment:
Likewise for `<FeaturedContent />`, `<TodayInHistory />`, `<SubscriptionBox />`. Seems like we can safely remove the styling on lines like 180, which were only needed because `CardContent` was missing.
https://github.com/ModularHistory/modularhistory/blob/c4f45f2c0ee06c82398330d7f6a554f548a1f855/frontend/pages/index.page.tsx#L179-L181